beta/src/components/DocsFooter.tsx

```diff
@@ -33,7 +35,7 @@ export const DocsPageFooter = memo<DocsPageFooterProps>(
                 <FooterLink
                   type="Previous"
                   title={prevRoute.title}
-                  href={removeFromLast(prevRoute.path, '.')}
+                  href={isExternal(prevRoute.path)}
```

Comment:
Did you check what `removeFromLast` does? Removing it might break some other thing?